packages/core/package.json

```diff
@@ -82,7 +85,6 @@
     "rimraf": "^5.0.5",
     "supertest": "^6.3.3",
     "tsc-alias": "^1.8.8",
-    "typescript": "^5.2.2",
```

Comment:
I noticed that the wagmi package.json still likes `typescript` (and `viem`) in dev deps. Not sure of the interaction between dev and peer deps but thought I'd mention. https://www.npmjs.com/package/wagmi?activeTab=code